src/components/play/questionTypes/vericalShuffle/VerticalShuffle.tsx

```diff
@@ -96,6 +97,13 @@ class VerticalShuffle extends CompComponent<VerticalShuffleProps, VerticalShuffl
     return this.state.userAnswers;
   }
 
+  prepareAttempt(component: VerticalShuffleComponent, attempt: ComponentAttempt<any>) {
```

Comment:
Can you undo this change for the moment because I just want to remove the attempt marks in this pull request.